### PR TITLE
first pass a2

### PR DESCRIPTION
**Pull request by brownsarahm**

### CODE REVIEW

**Review by ChamudiKashmila — APPROVED** [review]

File: assignments/02-python-access.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+````{margin}
+```{tip}
+Urls are strings. The `string` class in python has a lot of helpful methods for manipulating strings, like [`split`](https://docs.python.org/3/library/stdtypes.html#str.split).
+```
+````
+
+
+## Make a dataset about your datasets
+
+Import the list fomr the `datasets` module you created in the step above.
```
Comment (ChamudiKashmila) [inline assignments/02-python-access.md:71]: Import the list from the `datasets` module you created in the step above.

**Review by ChamudiKashmila — APPROVED** [review]

File: assignments/02-python-access.md (hunk 1 above)
Comment (brownsarahm) [inline assignments/02-python-access.md:71]:
```suggestion
Import the list from the `datasets` module you created in the step above.
```

File: assignments/02-python-access.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+For each dataset, in a separate section of your notebook titled `When things go wrong`:
+- try reading in data with the wrong `read_` function and make notes about what happens.
+- try to read in the `.csv` file that's included in the template repository (), use the error messages you get to try to fix the file manualy (any text editor, including jupyter can edit a `.csv`), making notes about what changes you made in a markdown cell.
+
+
+
+## Thinking ahead
+
+```{warning}
+his section is not required, but is intended to help you get started thinking
```
Comment (brownsarahm) [inline assignments/02-python-access.md:119]:
```suggestion
This section is not required, but is intended to help you get started thinking
```

File: assignments/02-python-access.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+about ideas for your portflio.  If you complete it, we'll give your feedback to
+help shape your ideas to get to level 3 achievements.  If you want to focus only
+on level 2 at this moment in time, feel free to skip this part.
+```
+
```
Comment (brownsarahm) [inline assignments/02-python-access.md:124]:
```suggestion
_answer these under a separate heading at the end of your notebook_
```

**Pull request merged**